### PR TITLE
New pricing tiers

### PR DESCRIPTION
**Pull request by sqs**
See gdoc at https://docs.google.com/document/d/1ybmvxonuhf4Y9pvU8BgcnAuu3emeHEdnqIBmg4XUj00/edit (private to Sourcegraphers only right now).

### CODE REVIEW

File: website/src/pages/pricing.tsx [hunk 1]
```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
...
                                 description="For individuals and small teams."
-                                price="Free"
+                                price="$0 (0-10 users), $10 (11-25 users)"
                                 priceInterval=""
                                 features={[
                                     { name: 'Code search', id: 'code-search' },
-                                    { name: 'Code navigation (definitions and references)', id: 'code-intelligence' },
-                                    { name: 'Editor and code host integrations', id: 'integrations' },
+                                    { name: 'Code navigation', id: 'code-navigation' },
+                                    { name: 'Code intelligence (25+ languages)', id: 'code-intelligence' },
```
Comment (christinaforney) [inline website/src/pages/pricing.tsx:50]: There are >30 languages :) See [3.13 blog](https://about.sourcegraph.com/blog/sourcegraph-3.13#basic-code-intelligence-support-reaches-32-languages)

```suggestion
                                    { name: 'Code intelligence (30+ languages)', id: 'code-intelligence' },
```

File: website/src/pages/pricing.tsx [hunk 2]
```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
             </Helmet>
             <div className="pricing-page">
                 <ContentSection color="primary" className="hero-section text-center py-5">
-                    <h1>Sourcegraph pricing</h1>
-                    <h3>
-                        Free to start. No commitment.
-                        <br />
-                        Only pay for active users.
-                    </h3>
+                    <h2>Universal Code Search, for less than the cost of one bad line of code.</h2>
```
Comment (dadlerj) [inline website/src/pages/pricing.tsx:35]: This is a little cheesy :). I think the line below is more effective (i..e, drop the "for less than" part)

File: website/src/pages/pricing.tsx [hunk 3]
```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
...
                 <div className="container-fluid pricing-page__plans">
                     <div className="row pt-6">
                         <div className="col-6 col-md-3 mx-auto mb-4">
                             <PricingPlan
                                 className="pricing__plan"
-                                name="Free"
+                                name="Starter"
                                 description="For individuals and small teams."
-                                price="Free"
+                                price="$0 (0-10 users), $10 (11-25 users)"
```
Comment (dadlerj) [inline website/src/pages/pricing.tsx:45]: @christinaforney when this is merged can we get help with automated billing for this? I'd really hate to have to go invoice a bunch of companies for $10. We have a lot of the infrastructure set up already (we had a full online self-serve billing at one point on an old pricing model), but someone has to go update our Stripe, make sure the billing UI works, that it correctly charges for each tier on the right periods, etc.

Comment (christinaforney) [inline website/src/pages/pricing.tsx:45]: Is the timeframe specified anywhere? If not, we should add "per year" and make it clear it is not per user but for the entire team < 25.

Comment (christinaforney) [reply website/src/pages/pricing.tsx:45]: Yes, absolutely! Do we have a sense of how many existing companies will be impacted by this change?

File: website/src/pages/pricing.tsx [hunk 4]
```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
...
                                 priceInterval=""
                                 features={[
                                     { name: 'Code search', id: 'code-search' },
-                                    { name: 'Code navigation (definitions and references)', id: 'code-intelligence' },
-                                    { name: 'Editor and code host integrations', id: 'integrations' },
+                                    { name: 'Code navigation', id: 'code-navigation' },
+                                    { name: 'Code intelligence (25+ languages)', id: 'code-intelligence' },
+                                    { name: 'Code change management campaigns', id: 'campaigns' },
```
Comment (christinaforney) [inline website/src/pages/pricing.tsx:51]: Are we going to specify the limit on changesets, or no limit at this time (should we indicate this)?

**Comment by sqs** [comment]
BTW this is a very early draft PR, not worth reviewing the diff yet (but feel free to leave a checklist of things you had in mind for when it’s ready)

**Comment by sqs** [comment]
Posted WIP screenshot at https://sourcegraph.slack.com/archives/C0B2RU51Q/p1586327721252800

File: website/src/pages/pricing.tsx (hunk 4 above)
Comment (sqs) [reply website/src/pages/pricing.tsx]: Yes, we will indicate this. See https://docs.google.com/document/d/1ybmvxonuhf4Y9pvU8BgcnAuu3emeHEdnqIBmg4XUj00/edit# for how I am planning on indicating it, and please leave comments there for now.

File: website/src/pages/pricing.tsx (hunk 3 above)
Comment (sqs) [reply website/src/pages/pricing.tsx]: I think this should be a one-time payment. Added to https://docs.google.com/document/d/1ybmvxonuhf4Y9pvU8BgcnAuu3emeHEdnqIBmg4XUj00/edit#, please comment there with your feedback.

**Comment by sqs** [comment]
Latest WIP design screenshot at https://sourcegraph.slack.com/archives/C0B2RU51Q/p1586407400287000.

File: website/src/components/pricing/PricingPlan.tsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    buttonOnClick,
+    buttonHref,
+}) => (
+    <div className={`pricing-plan card border-0 ${className}`}>
+        <h2 className="card-title border-bottom py-2 text-center">{name}</h2>
+        <div className="card-body pt-2 d-flex flex-column align-items-center flex-grow-0 text-center">
+            <div className="mb-3 pricing-plan__price">{price}</div>
+            {planProperties}
+        </div>
+        <ol className="list-group list-group-flush py-3">
```
Comment (aileenrose) [inline website/src/components/pricing/PricingPlan.tsx:120]: Repeat buttons closer to top of features list.
```suggestion
        <a  
            className="pricing-plan__button btn btn-primary justify-content-center text-center d-inline-flex"       
            href={buttonHref}
            onClick={buttonOnClick}
        >   
            {buttonLabel}
        </a>
        <ol className="list-group list-group-flush py-3">
```

File: website/src/pages/pricing.tsx (hunk 3 above)
Comment (sqs) [reply website/src/pages/pricing.tsx]: This $10 plan is not a part of this first stage of pricing, so not relevant now.

**Comment by sqs** [comment]
@aileenrose Great, I changed the button label to `Contact us`. I'm going to merge this. You can update the links to the new trial page when it's ready.

**Pull request merged**